stade/core/templates/base.html

```diff
@@ -99,6 +99,26 @@ <h1><a href="{% url 'index' %}"><img src="{% static 'img/isic_logo.svg' %}" alt=
     {% load invites %}
 
     {% show_pending_invites request %}
+
+    {% if not user.is_authenticated %}
+      <div class="has-text-centered">
```

Comment:
This is pretty intrusive at this location at the top of every page. Can we put it in the menu bar maybe?

Reply:
I'd rather add an x sign and save that in local storage.